src/plugins/vis_type_vislib/public/vislib/lib/types/point_series.js

```diff
@@ -21,7 +21,23 @@ import _ from 'lodash';
 import { i18n } from '@kbn/i18n';
 
 function getSeriId(seri) {
-  return seri.id && seri.id.indexOf('.') !== -1 ? seri.id.split('.')[0] : seri.id;
```

Comment:
I've tested the unit test with the previous implementation, they passed, also the float and not float result are identical?

Reply:
I know it wasn't covered before, but working on this a unit test for getSeriId would be 🌹

Reply:
You are quite right. The input JSON of the test was shaped in a way that the `data.series` was ignored for both scenarios. I've now pushed a newer versions of the tests where `data.series` is taken into account and the result JSON is now different. Also it fails with the previous version of the code and passes with the current one.